src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/ZipNameNotMatchMetaDataException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+public class ZipNameNotMatchMetaDataException extends InvalidEnvelopeException {
```

Comment:
either 'not match**ing**' or '**does** not match', I think.

Reply:
renamed as `ZipNameNotMatchingMetaDataException`